addons/main/functions/fnc_actionJamToggleListener.sqf

```diff
@@ -19,4 +19,12 @@ private _jammer = GVAR(jamMap) get _netId;
 _jammer set [3, _enabled];
 
 // save changes to map
-GVAR(jamMap) set [_netId, _jammer];
\ No newline at end of file
+GVAR(jamMap) set [_netId, _jammer];
+
+// show jammers in the spectrum
+if (_enabled) then {
+	[QEGVAR(spectrum,addBeacon), [_jamObj, 433, 300, "sweep"]] call CBA_fnc_globalEventJIP;
```

Comment:
I honestly don't know. 

I haven't tested anything with regards to the TFAR jammer. 
a) I don't know the exact functionality that it needs to have.
b) In order to test you need a bunch of people (which I never had available; nor a server were I could quickly change crowsEW).

I _assume_ it will behave as you wrote. 
In addition I made this comment in `fnc_spectrumTrackingLocal.sqf`.
```sqf
/*******************************************************************/
// TODO implement different frequency band for VoiceCommJammers
// (those are is currently shown in the drone band)
/*******************************************************************/
```

Reply:
Fair enough. Will test it out on a server. As I read it a jammer would always show a sweep now if enabled. Although that is also fine. Would make sense to see some kind of either sweep or just high noise broadcast for radio jamming.